liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -477,26 +448,10 @@ public static List<String> splitAndTrim(String s, String regex) {
         return returnList;
     }
 
-    /**
-     * @deprecated use {@link StringUtils#repeat(String, int)} instead
-     */
-    @Deprecated
-    public static String repeat(String string, int times) {
-        return StringUtils.repeat(string, times);
-    }
-
-    /**
-     * @deprecated use {@link StringUtils#join(Object[], String)} instead
-     */
-    @Deprecated
     public static String join(Integer[] array, String delimiter) {
         return StringUtils.join(array, delimiter);
     }
 
-    /**
-     * @deprecated use {@link StringUtils#join(int[], char)} instead
-     */
-    @Deprecated
```

Comment:
I think you should keep this deprecated message because there is a better option here, which is to use StringUtils.